etc/create_gluster_cluster_sample_job.py

```diff
@@ -0,0 +1,41 @@
+payload = {
+    "integration_id": "89604c6b-2eff-4221-96b4-e41319240240",
+    "run": "tendrl.flows.CreateCluster",
+    "type": "node",
+    "created_from": "API",
+    "created_at": "2017-03-09T14:15:14Z",
+    "username": "admin",
+    "parameters": {
+        "Node[]": ["81720b6c-6732-49dd-ad32-15845f199c79",
+                   "ab003bbc-00cd-4b74-b236-f19c2c33b96b"],
+        "sds_type": "glusterfs",
+        "sds_version": "3.2.1",
+        "name": "MyCluster",
+        "fsid": "140cd3d5-58e4-4935-a954-d946ceff371d",
+        "public_network": "10.70.40.0/22",
+        "cluster_network": "10.70.40.0/22",
+        "node_configuration": {
+            "ab003bbc-00cd-4b74-b236-f19c2c33b96b": {
+                "role": "glusterfs/node",
+                "provisioning_ip": "10.70.42.183",
```

Comment:
what is provisioning_ip here?
In gluster's case its mostly the management ip address.
@brainfunked , comments please